### PR TITLE
Update jquery-tooltipster to v4.0.0

### PR DESCRIPTION
**Pull request by louisameline**

### CODE REVIEW

File: files/jquery.tooltipster/update.json [hunk 1]
```diff
@@ -3,6 +3,6 @@
   "name": "jquery.tooltipster",
   "repo": "iamceege/tooltipster",
   "files": {
-	"include": ["./css/**/*", "./js/**/*"]
+	"include": ["./dist/**/*.min.js", "./dist/**/*.min.css"]
```
Comment (jimaek) [inline files/jquery.tooltipster/update.json]: Remove dist from here and use it as basePath parameter

Comment (louisameline) [inline files/jquery.tooltipster/update.json]: Thank you

File: files/jquery.tooltipster/info.ini [hunk 2]
```diff
@@ -1,5 +1,5 @@
 author = "Caleb Jacob"
 github = "https://github.com/iamceege/tooltipster/"
-homepage = "http://calebjacob.com/tooltipster/"
-description = "A powerful, flexible jQuery plugin enabling you to easily create semantic, modern tooltips enhanced with the power of CSS."
-mainfile = "js/jquery.tooltipster.min.js"
\ No newline at end of file
+homepage = "http://iamceege.github.io/tooltipster/"
+description = "A flexible and extensible jQuery plugin for modern tooltips."
+mainfile = "dist/js/tooltipster.bundle.min.js"
```
Comment (jimaek) [inline files/jquery.tooltipster/info.ini]: Remove `dist/` here and it should be good to go.

P.S. Comment to notify me of your changes.

**Comment by louisameline** [comment]
Updated, thank you

**Pull request merged**